linux/devdoc/file_util_linux_requirements.md

```diff
@@ -0,0 +1,66 @@
+# file_util_linux
+
+## Overview
+
+Linux implementation of the file operation functions.
+
+## Exposed API 
+
+```c
+// Most defines removed for space constraints
```

Comment:
Since INVALID_HANDLE_VALUE is the only value returned by the module, you can remove this comment to avoid confusion.